tsconfig.json

```diff
@@ -21,5 +21,6 @@
     "alwaysStrict": true,
     "noEmit": true
   },
-  "include": ["./src/**/*", "./*.js"]
+  "include": ["./src/**/*", "./*.js"],
+  "exclude": ["./node_modules/**/*"]
 }
```

Comment:
A good chunk of errors were based around `PACKAGE_NAME cannot be used as a JSX component`  or `NODE_MODULE PATH.ReactNode' is not assignable to type 'React.ReactNode'`. 

I found this [StackOverflow thread](https://stackoverflow.com/questions/71791347/npm-package-cannot-be-used-as-a-jsx-component-type-errors) but adding `resolutions` didn't work. I ended up deleting `yarn.lock` and reinstalling which cleared all the errors. I looked over the site and everything looks built correctly 🤔 But if anyone has a better idea to clearing the errors, please let me know!

```
node_modules/@dvcorg/gatsby-theme-iterative/src/components/MainLayout/index.tsx:59:9 - error TS2322: Type 'import("/home/circleci/repo/node_modules/@dvcorg/gatsby-theme-iterative/node_modules/@types/react/index").ReactNode' is not assignable to type 'React.ReactNode'.

node_modules/@dvcorg/gatsby-theme-iterative/src/components/Page/DefaultSEO/index.tsx:105:6 - error TS2786: 'Helmet' cannot be used as a JSX component.
  Its instance type 'Helmet' is not a valid JSX element.
```